package.json

```diff
@@ -492,7 +492,7 @@
     "eslint-import-resolver-typescript": "^2.5.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^26.6.0",
-    "eslint-plugin-jsdoc": "^39.3.3",
+    "eslint-plugin-jsdoc": "41.1.2",
```

Comment:
This isn't the latest version, but it IS the minimum version that works with typescript. If we try to update to latest versions of our eslint plugins we'll need to make thousands, maybe 10s of thousands, of changes.